doc/syslog/syslog-design.md

```diff
@@ -195,48 +244,70 @@ The `rsyslog-config` service performs the next actions:
 1. Renders `rsyslog.conf.j2` template with `sonic-cfggen` to generate a new `rsyslogd` config file
 2. Restarts `rsyslog` service which triggers `rsyslogd` to load a new config file
 
-### 2.3.2 SSIP parameters
+### 3.3.2. global syslog configuration
 
-**SSIP will have the next parameter mapping:**
+#### 3.3.2.1. Table 4: Global configuration parameters
+| SONiC                 | Rsyslogd | Config DB Schema                           |
+|:----------------------|:---------|:-------------------------------------------|
+| format                | template | SYSLOG_CONFIG\|GLOBAL\|format              |
+| trap                  | priority | SYSLOG_CONFIG\|GLOBAL\|trap                |
+| welf_firewall_name    | trap     | SYSLOG_CONFIG\|GLOBAL\|welf_firewall_name  |
+
+* trap will set rules trap severity if not specified in the rule
+
+#### 3.3.2.2. Format: `set`:
+
+Foramt:
+template(name="WelfFormat" type="string" string="%TIMESTAMP% %HOSTNAME% id=firewall time=\"%timereported\
+:::date-year%-%timereported:::date-month%-%timereported:::date-day% %timereported:::date-hour%:%timereported:::date-minute%:%timereported\
+:::date-second%\" fw=\"{{ firewall-name  }}\" pri=%syslogpriority% msg=\"%syslogtag%%msg:::sp-if-no-1st-sp%%msg:::drop-last-lf%\"\n")
+
+### 3.3.3. SSIP parameters
 
-###### Table 3: SSIP parameters
+**SSIP will have the next parameter mapping:**
 
-| SONiC  | Rsyslogd | Config DB Schema           |
-|:-------|:---------|:---------------------------|
-| key    | target   | SYSLOG_SERVER\|key         |
-| source | address  | SYSLOG_SERVER\|key\|source |
-| port   | port     | SYSLOG_SERVER\|key\|port   |
-| vrf    | device   | SYSLOG_SERVER\|key\|vrf    |
+#### 3.3.3.1. Table 5: SSIP parameters
+| SONiC         | Rsyslogd | Config DB Schema                   |
+|:--------------|:---------|:-----------------------------------|
+| key           | target   | SYSLOG_SERVER\|key                 |
+| source        | address  | SYSLOG_SERVER\|key\|source         |
+| port          | port     | SYSLOG_SERVER\|key\|port           |
+| vrf           | device   | SYSLOG_SERVER\|key\|vrf            |
+| protocol      | protocol | SYSLOG_SERVER\|key\|protocol       |
+| filter        | ereregex | SYSLOG_SERVER\|key\|filter         |
```

Comment:
filter values should be include/exclude right? The filter_pattern should be ereregex.

Reply:
correct, but from what i understood this table indicates the related parameter in the rsyslogd config file and this is why i wrote eregex. there is no include/exclude field there. it controls the "!" before eregrex field.
do you want me to change it?